docs/registration.md

```diff
@@ -40,29 +39,20 @@ You should receive an email at your registered address, with a verification link
+Once email verification is complete, you can now choose the roles you would like to have. All roles are tied to a resource server to which the role would be applicable to. For example:
+* Choosing the provider role for resource server `rs.example.com` would allow you to upload resources onto the `rs.example.com` resource server and manage access to said resources
+* Choosing the consumer role for resource server `rs.example.com` would allow you to request for access for resources belonging to `rs.example.com` and access data on said resource server for permitted resources
+
+**NOTE : If the consumer role is chosen, all resource servers are automatically selected and the user would get the consumer role for any new resource server that is added.**
 
 ![Choosing consumer role](../resources/auth/cons-role.png)<br/>
 *Choosing consumer role*
 
-### Provider and Delegate Registration
-
-Providers and delegates are required to be associated with an organization. In the second step of registration, when choosing those roles, it is necessary to also choose an organization when selecting either `provider` or `delegate` roles. 
-
-**The registered email domain needs to match the chosen organization's domain.** For e.g., *delegate@datakaveri.org* may be able to register as a delegate associated with Data Kaveri organization (as the domain is datakaveri.org), but not with IISc (as the domain is iisc.ac.in). 
-
-In case you plan to register for these roles, please choose the appropriate email address. Registering for the consumer role has no such restrictions.
-
-**NOTE: Provider registration is subject to approval. A user registered with the provider role may be able to access the Publisher Panel once they have been approved**.
-
-![Choosing provider/delegate role](../resources/auth/prov-dele.png)<br/>
-*Choosing provider/delegate role*
-
-## 4. Generate Client ID + Secret
+### Provider Registration
 
-Finally, after specifying the roles required in the previous step, an autogenerated client ID + secret is presented. These credentials can be used to request for tokens via scripts or on the command line.
+Users who require the provider role would need to give some extra information regarding the organization they are associated with. 
 
-**Note: The client secret can only be viewed here when it is first generated and will not be visible in the future, so please download the CSV file and keep it safe.**
+**NOTE: Provider registration is subject to approval. A user registered with the provider role may be able to access the Provider Panel once they have been approved by the admin of the Resource Server they have chosen**.
 
-![Successful registration; client ID and client secret](../resources/auth/success-reg.png)<br/>
-*Successful registration; client ID and client secret*
+![Choosing provider role](../resources/auth/prov-role.png)<br/>
+*Choosing provider role*
```

Comment:
This looks a bit open ended after 'COMPLETE YOUR PROFILE', can we add possibly two more screenshots to give it a closure:
1. Indicating the completed profile is pending to be approved by the IUDX admin. (We can also consider mentioning that the provider may connect with the point of contact in IUDX to expedite this.)
2. On approval, what screen a registered provider will be displayed(we may consider this for the consumer too) and a link to his next steps from here.

Please ignore the above if the doc flow already covers this.

Reply:
- I was able to add a screenshot to show the message displayed when a provider registers successfully
    - I was not able to do the same for consumers as the message was not shown long enough to take a screenshot
- I'm unable to show the registered provider's panel as I don't have credentials for a valid provider